crates/swift-bridge-cli/src/clap_app.rs

```diff
@@ -100,4 +100,16 @@ fn create_package_command() -> Command<'static> {
                 .required(true)
                 .help("The name for the Swift Package"),
         )
+        .arg(
+            Arg::new("resource")
+                .long("resource")
+                .takes_value(true)
+                .value_name("SRC_PATH:DEST_PATH")
+                .multiple_values(true)
+                .help(
```

Comment:
Can we update the book example to include bundling a resource?

https://github.com/chinedufn/swift-bridge/blob/2783798ef267135aeafdbb8981c7ff93efb0dfb8/book/src/building/swift-packages/README.md#L132-L171

Reply:
I added it to the end as it is not mandatory to add resources.